sdk/aqueduct/backend/api_client.py

```diff
@@ -255,7 +255,7 @@ def connect_integration(
         headers.update(
             {
                 "integration-name": name,
-                "integration-service": service,
+                "integration-service": service.value,
```

Comment:
@kenxu95 I found this was giving me an error saying that `service` was of type Enum so that couldn't be used in the headers. This ensures that the actual string value is passed along. I'm not sure why it wasn't causing errors earlier.